README.md

```diff
@@ -37,12 +37,13 @@ If you're working locally you can create a file called `.gitignore` at the root
 It's useful to declare dependencies within your project. This ensures that other people who clone your repo will be able to replicate your exact environment (since your node modules don't get synced with Github).
 
 There are two ways to save dependencies:
-+ `npm install <package-name> --save` (this is the default for `npm install` in npm5)
++ `npm install <package-name> --save` (this option is not necessary unless you are using a
+  version of npm older then 5.0, see the [npm documentation](https://docs.npmjs.com/cli/install))
```

Comment:
Could we clarify what me mean by "option"?

Maybe "the `--save` flag is no longer necessary, unless you're using a version of npm older than 5.0"

You've also got a typo: "then" -> "than"